Profiler/TriggerProfiler.md

```diff
@@ -0,0 +1,61 @@
+## Diagnosing a WebAPI experiencing intermittent high CPU using the Application Insights Profiler
```

Comment:
nit: I would start with single `#` because that's head level 1. I would not try to consider too much about the formatting since we don't control the rendering of markdown.